angular-notify.js

```diff
@@ -57,19 +66,28 @@ angular.module('cgNotify', []).factory('notify',['$timeout','$http','$compile','
                 angular.element(args.container).append(templateElement);
                 messageElements.push(templateElement);
 
-                if (args.position === 'center'){
-                    $timeout(function(){
-                        templateElement.css('margin-left','-' + (templateElement[0].offsetWidth /2) + 'px');
-                    });
-                }
+                $timeout(function(){
+                    switch (args.position){
+                        case 'center':  templateElement.addClass('cg-notify-message-center');
```

Comment:
I would much rather use ng-class here and just put the position property on scope.  This would allow folks who override the template to be able to customize the positioning.  Perhaps they want a different margin when they position left or right.  The centering still will require the special offseting margin-left on the templateElement.